workshops/Optimized_TF/README.md

```diff
@@ -0,0 +1,134 @@
+# Install instructions for GPU use
+
+1. Navigate to your eagle home directory or scratch directory using
+```
+cd ~/
+```
+or
+```
+cd /scratch/$USER/
+```
+2. Clone the github repo into the directory you chose
+```
+git clone https://github.nrel.gov/hpc-apps/Optimized_TF
+```
+3. Navigate to the repo
+```
+cd ./Optimized_TF/
```

Comment:
update based on the different repo being cloned

Reply:
```suggestion
cd ./HPC/workshops/Optimized_TF/
```